modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -608,4 +637,46 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
 
         ctx.close();
     }
+
+    @Override
+    public CompletableFuture<?> onUpdate(ConfigurationNotificationEvent<AuthenticationView> ctx) {
+        return CompletableFuture.runAsync(() -> {
```

Comment:
Is there a reason to use `runAsync`? Looks like we can run the code synchroinously.